components/hero/Hero.tsx

```diff
@@ -0,0 +1,110 @@
+import Image from 'next/image';
+
+import Divider from '@mui/material/Divider';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+
+import UpworkIcon from '../../components/common/icon-upwork/UpworkIcon';
+import ClutchIcon from '../../components/common/icon-clutch/ClutchIcon';
+import Button from '../../components/common/buttons/Button';
+import BoxIcon from '../../components/common/icon-box/BoxIcon';
+import TriangleIcon from 'components/common/icon-triangle/TriangleIcon';
+import RectangleLeftIcon from '../../components/common/icon-rectangle-left/RectangleLeftIcon';
+import RectangleRightIcon from '../../components/common/icon-rectangle-right/RectangleRightIcon';
+
+import placeholderLogo from '../../public/PlaceholderLogo.png';
+import placeholderLogo1 from '../../public/PlaceholderLogo1.png';
+import placeholderLogo2 from '../../public/PlaceholderLogo2.png';
+
+import {
+  UpdateLabel,
+  PageContainer,
+  FixedBottomContainer,
+  ScrollContainer,
+  RectangleLeftIconContainer,
+  RectangleRightIconContainer,
+  BoxIconContainer,
+  TriangleIconContainer,
+  TitleTypography,
+  SubTitleTypography,
+  FirstItem,
+} from './styled-components';
+
+const Hero = () => {
+  return (
+    <PageContainer>
+      <RectangleLeftIconContainer>
+        <RectangleLeftIcon />
+      </RectangleLeftIconContainer>
+      <RectangleRightIconContainer>
+        <RectangleRightIcon />
+      </RectangleRightIconContainer>
+      <Container
+        maxWidth="lg"
+        sx={{
+          position: 'relative',
+          marginBottom: '116px',
+          paddingTop: { xs: '160px', lg: '270px' },
+        }}
+      >
+        <BoxIconContainer>
+          <BoxIcon />
+        </BoxIconContainer>
+        <TriangleIconContainer>
+          <TriangleIcon />
+        </TriangleIconContainer>
+        <Typography align="center" sx={{ marginBottom: { xs: 3.5, md: 2 } }}>
+          <UpdateLabel>Update</UpdateLabel> Data export & import, relations reordering, audit logs{' '}
+          <Link href="#" variant="body1">
```

Comment:
As far as I see, this type of link should have `font-weight: 500`

Reply:
'+